src/components/AgmExtension/ExtensionRequest.vue

```diff
@@ -11,9 +11,12 @@
     <template #content>
       <div
         class="px-6 py-7"
-        :class="{ 'invalid-section': !extensionRequestValid && showErrors }"
+        :class="{ 'invalid-section': !firstSectionValid && showErrors }"
```

Comment:
Added specific valid flags for each section of the component to highlight specific areas that are not valid.